src/legacy/ui/public/vis/editors/default/schemas.d.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export type Schemas = any;
```

Comment:
It's not safety to use `any` type. And while we're on `Schemas`, let's describe a type of `Schema` since `Schemas` is an array of `Schema` object. 
For example it may look like this:
```ts
import { AggParam } from '../../../agg_types';

export interface Schema {
  aggFilter: string | string[];
  deprecate: boolean;
  editor: boolean | string;
  group: 'metrics' | 'buckets';
  max: number;
  min: number;
  name: string;
  params: AggParam[];
  title: string;
}
```

Reply:
Thanks, done